### ISSUE TITLE
Does this support 1.21.1?

### ISSUE THREAD

**Issue by ColdSight** [Hadimhz/WelcomeBack#5]
Currently running 1.21.1 and it does not know if a person has joined or not (Reason why is because it says the welcome party has expired even though the player just joined).

**Comment by Hadimhz** [comment]
Yeah, updating to the latest on my server seems to cause the same issue too, I will be looking into it later today, I will make sure to keep you updated <3

**Comment by ColdSight** [comment]
Sounds good

**Comment by ColdSight** [comment]
Any updates?

**Comment by Hadimhz** [comment]
@ColdSight This has been fixed, and i've sent it to a few people with a similar problem in order to confirm I will let you know when they get back to me
